base/cvd/BUILD.libsrtp2.bazel

```diff
@@ -56,15 +56,6 @@ cc_library(
         "include/srtp_priv.h",
         "include/ut_sim.h",
     ],
-    copts = [
-        "-fPIC",
-        "-Wall ",
-        "-pedantic",
-        "-Wstrict-prototypes",
```

Comment:
If I understand correctly these are the only `copts` that are related to warnings, while `-fPIC` is related to linking, `-O4` and `-funroll-loops` are performance. Not sure what `-c` is doing there.

Reply:
That's fair, I imagine a lot of specific compiling optimization flags are "this made it faster on my machine once 10 years ago" at best.